### ISSUE TITLE
Move MP Intro topic to appear with other docs

### ISSUE THREAD

**Issue by lauracowen** [OpenLiberty/openliberty.io#1311]
Currently, the [MP Intro topic](https://github.com/OpenLiberty/docs/blob/master/intro/microprofile.adoc) is not displayed with the other docs; it's just linked to from the [guides page](https://openliberty.io/guides/).

Can we move it to be alongside the other doc topics (so it appears on the [docs page](https://openliberty.io/docs/ref/general/))? This would also require a redirection from the old URL to the new URL so we don't break links (@NottyCode implemented a redirection capability for this).

**Comment by lauracowen** [comment]
(I've moved to the openliberty.io repo because it relates to the website infrastructure.)

**Comment by steven1046** [comment]
Do we want it to be in the general references as well as linked to from where the link is at the moment?

**Comment by lauracowen** [comment]
Yes, we want it to just be a doc topic like the others but still have a link to it from the Guides page.

**Comment by steven1046** [comment]
Ok!
